databases/week2/homework-part-3.sql

```diff
@@ -0,0 +1,36 @@
+USE hyf_lesson2;
+
+-- Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT task.title, user.name, user.email
+FROM task
+JOIN user_task
+ON task.id = user_task.task_id
+JOIN user
+ON user_task.user_id = user.id
+WHERE user.email LIKE "%@spotify.com";
+	
+-- Get all the tasks for 'Donald Duck' with status 'Not started
+SELECT user.name, task.title, status.name
+FROM task
+JOIN user_task
+ON task.id = user_task.task_id
+JOIN status
+ON task.status_id = status.id
+JOIN user
+ON user_task.user_id = user.id
+WHERE user.name = "Donald Duck" AND status.name = "Done";
+
+-- Get all the tasks for 'Maryrose Meadows' that were created in september (hint: month(created)=month_number)
+SELECT user.name, task.title, task.created
+FROM user_task
+JOIN task
+ON user_task.task_id = task.id
+JOIN user
+ON user_task.user_id = user.id
+WHERE user.name = "Maryrose Meadows" AND month(task.created) = 9 AND MONTH;
```

Comment:
This query gives me an error: `Error Code: 1054. Unknown column 'MONTH' in 'where clause'`